package.json

```diff
@@ -181,6 +181,11 @@
 					"type": "string",
 					"default": null,
 					"description": "Path to Chrome executable for browser use functionality. If not set, the extension will attempt to find or download it automatically."
+				},
+				"cline.editor.skipDiffAnimation": {
+					"type": "boolean",
+					"default": true,
```

Comment:
The default here should be false.  I know that this is counter intuitive, but if this defaults to true, you current code eliminates the animation completely.  This should be a feature to be turned off rather than never thought to be turned on.